release-notes/2024-12/README.md

```diff
@@ -0,0 +1,198 @@
+# SystemLink Enterprise 2024-12 Release Notes
+
+The 2024-12 release for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Behavior change or new feature description
+
+- Updated **Automated Agent** built-in role to include privileges to **Execute test actions**.
+
+## Helm Chart Breaking Changes
+
+- `dynamicformfields:0.1.310`
+    - The DynamicFormFields (DFF) Service was added to the top-level chart.
+        - [View this service configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-12/getting-started/templates/systemlink-values.yaml#L1260)        - [View this service configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-12/getting-started/templates/systemlink-values.yaml#L1260)
+    - Secrets for the service's Postgres credentials need to be configured in systemlink-secrets.yaml.
```

Comment:
```suggestion
    - You must configure the secrets for the PostgreSQL credentials of the DFF service in systemlink-secrets.yaml.
```

Not sure if this is for the DFF service or another service. Either way, we should be specific.